### DISCUSSION TITLE
How do I "unwatch" a repo?

### DISCUSSION THREAD

**Discussion by Aviation6** [community/community#35086] [New to GitHub] [answered]
I'm pretty new to the Git workflow so I don't know how to unwatch a repo as there isn't a button that says "unwatch"

I'd really appreciate it if someone told me how silly I am being and what obvious button I need to click

Thanks

**Comment by antonblr** [comment]
You should be able to  "unwatch" in the upper-right corner - "Watch" drop down settings by selecting "Ignore":

<img width="600" alt="Screen Shot 2022-10-04 at 8 12 43 AM" src="https://user-images.githubusercontent.com/44246099/193857837-89929d81-4b37-43bd-9a7f-e93b5eed37c5.png">

**Reply by mgjv** [reply]
ignoring notifictions isn't the same as not watching it.

**Reply by rosaboyle** [reply]
Exactly

**Comment by Belikeoscar** [comment, marked as answer]
You can simply unwatch by clicking on the unwatch drop down at the top corner, it would probably have the All activity option ticked so just select the Participating and @mentions option and you would be clear.

**Reply by Arithkauv** [reply]
Unwatch Button is missing.
<img width="1345" height="534" alt="image" src="https://github.com/user-attachments/assets/0c93792e-9fcd-4387-8f09-c31c36e34336" />

**Comment by official-mywebprojects** [comment]
To unwach a repo, In the left sidebar, under the list of repositories, use the "Manage notifications" drop-down to click Watched repositories. On the watched repositories page, after you've evaluated the repositories you're watching, choose whether to: Unwatch a repository.

More from this link: https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/managing-your-subscriptions

**Comment by techemmy** [comment]
To unwatch the repo, just select the Participating and @mentions option on that dropdown

**Reply by Kazuto** [reply]
Thanks. Github could make it more clear that this option unwatches a repo.

**Reply by npearson72** [reply]
Hilarious! The amount of time I spent running in circles trying to just remove some old work repos. 😆

**Reply by rosaboyle** [reply]
Exactly

**Reply by amanharshx** [reply]
omg thanks.

**Reply by primeminister** [reply]
But it does not remove the repo from your 'Watched repositories'. Bad UI.

**Comment by Peter-Two-Point-O** [comment]
Is there no way to permanently unwatch a repo?  The current method of simply selecting "ignore" still has the repo show up on your watch list.

**Reply by techemmy** [reply]
On the `Watch` dropdown, Click on participating and @mentions. That should unwatch a repo permanently.

**Reply by ahmedam55** [reply]
Thanks for the help!

**Reply by oldman20** [reply]
> On the `Watch` dropdown, Click on participating and @mentions. That should unwatch a repo permanently.

Many thank, bad design, dark/ stupid UI

**Reply by yuseok** [reply]
> On the `Watch` dropdown, Click on participating and @mentions. That should unwatch a repo permanently.

This is exactly what I wanted! Thank you!

**Reply by Fuwachiro** [reply]
what the heck! thats exactly what i want! why it named "participating and @mentions" when it should just say "Unwatch" instead?

**Comment by rebort575** [comment]
what a bad design, bothered me for a long time.

**Reply by 127** [reply]
Shitty UX to be honest

**Reply by lgy** [reply]
I think github do this on purpose, it was named "unwatch" several years ago, but they changed it in order get more attention for repositories, when many people choose to ignore, it's still showing them watching, this is crazy for people who don't want to be seen, for example on some copycats project someone finally notice they are being fooled. Also is causing trouble for people like me who wants a clean watching list. The standard way is to name those options such as "watch", "unwatch", "block all" etc., especially when you block it it shouldn't appear on your watching list unlike this "ignore" button. Current naming and behavior drive me crazy for months. Thanks for people who inserted this post and replied this question.

**Comment by Med-Amine-AB** [comment]
i have several repositories are appearing on my dashboard even though I have taken steps to remove them. like unwatch and unsubscribe

**Reply by Martin-Dolan** [reply]
go to the watching/unwatching/ignore dropdown thing and select "Participating and @mentions"

that means you're unwatched/unsubscribed..    I know...  it doesn't make sense, but that's how you do it.

It means you'll only get notified if you participate (which you're not) or if someone @mentions you...  which they wont.  
so basically if I went to *any* repo and @mentioned you, you would get notified (Unless you are specifically ignoring that repo)  but the default ignore state for any repo is  "Participating and @mentions" which means you're not watching it...

hope this explanation helps anyone else out who finds this...

**Reply by Alek2077** [reply]
This is the solution.

**Comment by SilverPaladin** [comment]
I have tried to click on the Participating and @mentions but the check mark never goes away! I cannot unwatch a repo it seems.

**Reply by techemmy** [reply]
You don't have to worry about that. The "Participating and @mentions" is the default. You can't uncheck it except you choose another option in the dropdown.

It simply means you'll only get notified if someone mentions you (which is unlikely if they don't know you) or if you perform any actions that require GitHub to notify you.

**Comment by Arcadi4** [comment]
Have they changed this feature? Now the unwatch button in <https://github.com/watching> just does not work.

**Reply by Fuwachiro** [reply]
nope, it works on my side.
